project-base/src/Shopsys/ShopBundle/Resources/views/Front/Content/ContactForm/index.html.twig

```diff
@@ -0,0 +1,54 @@
+{% extends '@ShopsysShop/Front/Layout/layoutWithPanel.html.twig' %}
+
+{% block title %}
+    {{ 'Contact'|trans }}
+{% endblock %}
+
+{% block main_content %}
+    <div class="h2">{{ 'Contact us'|trans }}</div>
+
+    <div class="in-user-text margin-bottom-20">
+        {{ mainText|raw }}
+    </div>
+
+    {{ form_start(form) }}
+
+    {{ form_errors(form) }}
+
+        {{ form_row(form.name, { label: 'Full name'|trans }) }}
+        {{ form_row(form.email, { label: 'Email'|trans }) }}
+        {{ form_row(form.message, { label: 'Message'|trans }) }}
+        <dl class="form-line">
+            <dt></dt>
+            <dd>
+                <div class="form-choice">
+                    {% if privacyPolicyArticle is not null %}
+                        {% set articleUrl = url('front_article_detail', {'id': privacyPolicyArticle.id}) %}
+                        {{ form_widget(form.privacyPolicy, { attr: { class: "css-checkbox" }} )}}
+                        <label for="{{ form.privacyPolicy.vars.id }}" class="css-checkbox__image">
+                            {% if privacyPolicyArticle is not null %}
+                                {% set articleUrl = url('front_article_detail', {'id': privacyPolicyArticle.id}) %}
+                                {{ 'I agree with <a href="%url%" target="_blank">privacy policy</a>.'|trans({'%url%': articleUrl})|raw }}
+                            {% else %}
+                                {{ 'I agree with privacy policy.'|trans }}
+                            {% endif %}
+                        </label>
+                    {% else %}
+                        {{ form_widget(form.privacyPolicy, { attr: { class: "css-checkbox" }} )}}
+                        <label for="{{ form.privacyPolicy.vars.id }}" class="css-checkbox__image">
+                            {{ 'I agree with privacy policy.'|trans }}
+                        </label>
+                    {% endif %}
```

Comment:
In registration is this block like 
```twig
{{ form_widget(form.privacyPolicy, { attr: { class: "css-checkbox" } }) }}
    <label for="{{ form.privacyPolicy.vars.id }}" class="css-checkbox__image">
        {% if privacyPolicyArticle is not null %}
            {% set articleUrl = url('front_article_detail', {'id': privacyPolicyArticle.id}) %}
            {{ 'I agree with <a href="%url%" target="_blank">privacy policy</a>.'|trans({'%url%': articleUrl})|raw }}
        {% else %}
            {{ 'I agree with privacy policy.'|trans }}
        {% endif %}
    </label>
```

which I found a little bit easier to read. What do you think? I noticed the condition for privacyPolicyArticle and url generation is twice in a block ;)

Reply:
Niiiiiiiiiiiiiiiiiiiice, fixed